ios/FluentUI/Notification/MSFNotification.swift

```diff
@@ -26,6 +26,14 @@ import UIKit
         notification = FluentNotification(style: style,
                                           shouldSelfPresent: false)
         super.init(AnyView(notification))
+        notification.state.actionButtonAction = style.shouldAlwaysShowActionButton
```

Comment:
nit: Would it be possible to store the logic after ? and make this a one-line statement for readability?